src/main.rs

```diff
@@ -0,0 +1,46 @@
+#![recursion_limit = "1024"]
+
+#[macro_use]
+extern crate error_chain;
+extern crate glib;
+#[macro_use]
+extern crate log;
+
+use glib::{KeyFile, KeyFileFlags};
```

Comment:
Not attached to the GLib keyfile format here - could switch this to use a TOML format for the `client.conf` file instead (either in this PR or later).

Reply:
I think it's indeed a better idea to just use TOML since the beginning.

Reply:
:+1: - fixed.